airflow/provider_info.schema.json

```diff
@@ -27,6 +27,27 @@
       "items": {
         "type": "string"
       }
+    },
+    "secrets-backends": {
```

Comment:
Can you update the user docs also?
http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow-providers/howto/create-update-providers.html

Reply:
I deliberately did not add it now. 

I plan to do it in the follow-up documentation-specific PR where I am going to generate automated "summaries" in providers and provide all the inter-linking from relevant parts of the documentation.